src/v2/Apps/ViewingRoom/Components/ViewingRoomsLatestGrid.tsx

```diff
@@ -88,14 +88,16 @@ export const ViewingRoomsLatestGrid: React.FC<ViewingRoomsLatestGridProps> = pro
               distanceToClose,
               artworksConnection,
             } = vr
-            const heroImageURL = crop(image?.imageURLs?.normalized, {
-              height: 800,
-              width: 800,
+            const heroImageURL = cropped(image?.imageURLs?.normalized, {
+              height: 490,
+              width: 490,
             })
             const artworksCount = artworksConnection.totalCount
-            const artworkImages = artworksConnection.edges.map(({ node }) =>
-              artworksCount < 2 ? node.image.regular : node.image.square
-            )
+            const artworkImages = artworksConnection.edges.map(({ node }) => {
+              const src =
+                artworksCount < 2 ? node.image.regular : node.image.square
+              return { src, srcSet: src }
```

Comment:
(nonblocking) This is a little confusing at a glance -- maybe we should update the refs  so that things match 1:1?

Reply:
Yeah, it looks like these should just be square crops anyway rather than trying to use these image versions.